DXRModules/DeusEx/Classes/DXRStartMap.uc

```diff
@@ -69,6 +82,16 @@ function PreFirstEntry()
         }
         break;
 
+    case "06_HongKong_Helibase":
+        if (dxr.flags.settings.starting_map > 60) {
+            RemoveJock('chopper');
```

Comment:
also needs to set the flags for MS_DoorsBlown and MS_HelicopterFired

I'm also still hearing helicopter sounds

and there's a leftover mover `jockweapons` blocking the player when they walk where Jock would have been 

![image](https://github.com/user-attachments/assets/70db48ed-a57b-4cc7-9c88-5d8b7d1a3fd6)